src/libAtomVM/otp_crypto.c

```diff
@@ -0,0 +1,593 @@
+/*
+ * This file is part of AtomVM.
+ *
+ * Copyright 2023 Paul Guyot <pguyot@kallisys.net>
```

Comment:
I think here `Copyright 2022 Fred Dushin <fred@dushin.net>` for hash code, and `Copyright 2023 Davide Bettio <davide@uninstall.it>` for symmetric crypto should be added.

Reply:
The code in otp_crypto.c is coming form `src/platforms/esp32/components/avm_sys/platform_nifs.c` indeed, so let's copy the copyright notice from there.